src/ui/public/error_allow_explicit_index/error_allow_explicit_index.html

```diff
@@ -1,26 +1,39 @@
 <div class="app-container kbnError--multi-allow-explicit-index euiText">
-  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon> Oh no!</h3>
+  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon>
+    <span i18n-id="common.ui.errorAllowExplicitIndex.errorTitle"
+    i18n-default-message="Oh no!"></span>
+  </h3>
 
-  <p>
-    It looks like your Elasticsearch cluster has the
-    <code>rest.action.multi.allow_explicit_index</code> setting set to <code>false</code>, which
+  <p i18n-id="common.ui.errorAllowExplicitIndex.errorDescription"
+    i18n-default-message="It looks like your Elasticsearch cluster has the
+    {allowExplicitIndexConfig} setting set to {allowExplicitIndexValue}, which
     prevents Kibana from making search requests. We use this ability to send a single request to
     Elasticsearch that searches multiple indexes so that when there are many panels on a dashboard
-    they will load quickly and uniformly.
-  </p>
+    they will load quickly and uniformly."
+    i18n-values="{
+      html_allowExplicitIndexConfig: '<code>rest.action.multi.allow_explicit_index</code>',
+      html_allowExplicitIndexValue: '<code>false</code>'
+    }"
+  ></p>
 
-  <p>
-    Unfortunately, until this issue is fixed you won't be able to use certain apps in Kibana, like
-    Discover, Visualize and Dashboard.
-  </p>
+  <p i18n-id="common.ui.errorAllowExplicitIndex.errorDisclaimer"
+    i18n-default-message="Unfortunately, until this issue is fixed you won't be able to use certain apps in Kibana, like
+    Discover, Visualize and Dashboard."
+  ></p>
 
-  <h3>Ok, how do I fix this?</h3>
+  <h3 i18n-id="common.ui.errorAllowExplicitIndex.fixErrorTitle"
+    i18n-default-message="Ok, how do I fix this?"
+  ></h3>
   <ol>
-    <li>
-      Remove <code>rest.action.multi.allow_explicit_index: false</code> from your Elasticsearch
-      config file.
-    </li>
-    <li>Restart Elasticsearch.</li>
-    <li>Use the browser's back button to return to what you were doing.</li>
+    <li i18n-id="common.ui.errorAllowExplicitIndex.fixErrorListItemOne"
```

Comment:
please do not use ItemOne, ItemTwo etc in id names. Instead add more descriptive id which describes message it is created for.